docs/docs/misconfiguration/custom/schema.md

```diff
@@ -0,0 +1,77 @@
+# Input Schema
+
+## Overview
+Policies can be defined with custom schemas that allow inputs to be verified against it. Adding an policy schema
+enables Trivy to show more detailed error messages when an invalid input is encountered.
+
+For instance if we have a policy that checks for misconfigurations in a `Dockerfile`, we could define the
```

Comment:
Dockerfile is a great example for input schemas because dockerfiles are not json compatible by nature and without json schema it is impossible to know what you are even evaluating. I think it's worth mentioning and even reframing the intro to emphasize this.

Reply:
added more context to show the benefit of input schemas